docs/coordinates/galactocentric.rst

```diff
@@ -0,0 +1,108 @@
+.. _coordinates-galactocentric:
+
+==========================================
+Transforming to Galactocentric coordinates
+==========================================
+
+[sec:transform]
+
+We assume that we start with a 3D position in the ICRS reference frame:
+a Right Ascension, Declination, and heliocentric distance,
+:math:`(\alpha, \delta, d)`. We can trivially convert this to a
+Cartesian position using the standard transformation from Cartesian to
+spherical coordinates:
+
+.. math::
+
+   \begin{aligned}
+       x_{\rm icrs} &= d\cos{\alpha}\cos{\delta}\\
+       y_{\rm icrs} &= d\sin{\alpha}\cos{\delta}\\
+       z_{\rm icrs} &= d\sin{\delta}\\
+       \boldsymbol{r}_{\rm icrs} &= \begin{pmatrix}
+         x_{\rm icrs}\\
+         y_{\rm icrs}\\
+         z_{\rm icrs}
+       \end{pmatrix}\end{aligned}
+
+The first transformations will rotate the :math:`x_{\rm icrs}` axis so
+that the new :math:`x'` axis points towards the Galactic Center (GC),
+specified by the ICRS position
+:math:`(\alpha_{\rm GC}, \delta_{\rm GC})`:
+
+.. math::
+
+   \begin{aligned}
+       \boldsymbol{R}_1 &= \begin{bmatrix}
+         \cos\delta_{\rm GC}& 0 & -\sin\delta_{\rm GC}\\
+         0 & 1 & 0 \\
+         \sin\delta_{\rm GC}& 0 & \cos\delta_{\rm GC}\end{bmatrix}\\
+       \boldsymbol{R}_2 &=
+       \begin{bmatrix}
+         \cos\alpha_{\rm GC}& \sin\alpha_{\rm GC}& 0\\
+         -\sin\alpha_{\rm GC}& \cos\alpha_{\rm GC}& 0\\
+         0 & 0 & 1
+       \end{bmatrix}.\end{aligned}
+
+The transformation thus far has aligned the :math:`x'` axis with the
+vector pointing from the Sun to the GC, but the :math:`y'` and
+:math:`z'` axes point in an arbitrary direction. We adopt the
+orientation of the Galactic plane as the normal to the north pole of
```

Comment:
Can you make `Blaauw et. al. 1960` a link that takes me to ADS? I'd find that convenient...